src/app/docs/alert/alert-docs.component.ts

```diff
@@ -2,8 +2,44 @@ import {
   Component
 } from '@angular/core';
 
+import {
+  SkyDocsDemoControlPanelChange,
+  SkyDocsDemoControlPanelRadioChoice
+} from '@skyux/docs-tools';
+
 @Component({
   selector: 'app-alert-docs',
   templateUrl: './alert-docs.component.html'
 })
-export class AlertDocsComponent { }
+export class AlertDocsComponent {
+  public typeChoices: SkyDocsDemoControlPanelRadioChoice[] = [
+    { value: 'danger', label: 'Danger' },
+    { value: 'info', label: 'Info' },
+    { value: 'success', label: 'Success' },
+    { value: 'warning', label: 'Warning' }
+  ];
+
+  public alertMessage = 'Danger alert message';
+
+  public demoSettings: any = {};
+
+  public onDemoSelectionChange(change: SkyDocsDemoControlPanelChange): void {
+    this.demoSettings.closed = false;
+
+    if (change.closable !== undefined) {
```

Comment:
Same question about `closeable` here. ... I changed `closable` to `closeable` in all instances in my branch after talking to Steve, and everything seems to work fine that way. But let me know if you see a problem with that.